Jellyfin.Plugin.PushOverNotifications/Api/ServerApiEntryPoints.cs

```diff
@@ -21,11 +20,11 @@ public class TestNotification : IReturnVoid
     class ServerApiEndpoints : IService
     {
         private readonly IHttpClient _httpClient;
-        private readonly ILogger _logger;
+        private readonly ILogger IloggerFactory;
 
-        public ServerApiEndpoints(ILogManager logManager, IHttpClient httpClient)
+        public ServerApiEndpoints(ILoggerFactory logManager, IHttpClient httpClient)
```

Comment:
What I meant here is ILoggerFactory loggerFactory.
Optionally if ServerApiEndpoints is created by some other part of this plugin, to pass down that logger instead of using a logger factory here.

Reply:
Unfortunately, ServerApiEndpoints is not created by any other part of the plugin, but by Jellyfin itself, to add a web API endpoint/entrypoint (terminology seems a bit inconsistent).

I've tested and it's possible to add an ILoggerFactory parameter to the Plugin constructor in Plugin.cs, which could then be used to create a logger that is shared with all the other components in the plugin, e.g. through Plugin.Instance.Logger. Not sure if that would be good practice...

... actually, I just checked the anime plugin, it just has an ILogger parameter in its Plugin constructor. Would be nice to decide on a standard way of getting a logger for a plugin...

Reply:
At least from my understanding it would be best if a plugin took an iloggerfactory in the plugin constructor, created a logger and passed it throughout the plugin

Reply:
But it's enough for me to just rename the variable and mark this as closed. The best practice should ideally be written in the jellyfin-docs